pkg/envoy/rds/new_response_test.go

```diff
@@ -34,6 +34,8 @@ func TestNewResponse(t *testing.T) {
 	certSerialNumber := certificate.SerialNumber("123456")
 	testProxy := envoy.NewProxy(certCommonName, certSerialNumber, nil)
 
+	testOutbound := []*trafficpolicy.OutboundTrafficPolicy{}
```

Comment:
We should probably have a scenario where there are some outbound traffic policy here and then some assertions below to show we're programming outbound virtual host correctly.

Reply:
we can do this in a follow up

Reply:
Yup, I have made a note of the same. Will have a PR dedicated to unit tests once we have all the functionality for routes v2